account-kit/react/src/components/auth/card/footer/help-text.tsx

```diff
@@ -2,7 +2,8 @@ import { useUiConfig } from "../../../../hooks/useUiConfig.js";
 import { ls } from "../../../../strings.js";
 
 export const HelpText = () => {
-  const { supportUrl } = useUiConfig();
+  const { supportUrl } = useUiConfig(({ supportUrl }) => ({ supportUrl }));
```

Comment:
most of these changes for the `useConfig` are done so that components subscribe just to the state they care about. This results in WAY fewer renders